staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -211,6 +211,14 @@ const (
 	//
 	// Allow the API server to stream individual items instead of chunking
 	WatchList featuregate.Feature = "WatchList"
+
+	// owner: @saschagrunert
+	// alpha: v1.28
+	//
+	// Enables user namespace support for Pod Security Standards. Enabling this
+	// feature will modify all Pod Security Standard rules to allow setting:
+	// spec[.*].securityContext.[runAsNonRoot,runAsUser,allowPrivilegeEscalation,capabilities]
+	UserNamespacesPodSecurityStandards featuregate.Feature = "UserNamespacesPodSecurityStandards"
```

Comment:
naively  would expect this feature gate to be the same that's defined in the top level. Is there a reason or precedent to add a new one? I didn't see discussion in the KEP about it

Reply:
Yes, this API server feature gate is separate from the other feature gate and only meant to be used temporarily, tt will never graduate.

cc @mrunalp